### PR TITLE
Add postinstall script to install all dependencies

### PR DESCRIPTION
**Pull request by yoyo930021**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "version": "0.25.0",
   "publisher": "octref",
   "scripts": {
+    "postinstall": "cd server && yarn && cd ../vti && yarn && cd ../",
```
Comment (octref) [inline package.json:10]: I'd say just use `(cd server && yarn) && (cd vti && yarn)`. It's cleaner.

Comment (yoyo930021) [reply package.json]: Done.

Comment (yoyo930021) [reply package.json]: @octref this script can run in windows?

Comment (octref) [reply package.json]: This won't run on Windows. I suggest that we use https://github.com/mysticatea/npm-run-all

Comment (octref) [reply package.json]: You need to update https://github.com/vuejs/vetur/blob/master/.travis.yml and https://github.com/vuejs/vetur/blob/master/azure-pipelines.yml as well.

Comment (yoyo930021) [reply package.json]: I'm done.

File: test/lsp/features/completion/template.test.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { activateLS, showFile, sleep, FILE_LOAD_SLEEP_TIME } from '../../helper';
```
Comment (octref) [inline test/lsp/features/completion/template.test.ts:1]: I know it's tempting to do these all together, but please keep each PR focused on one single task. For 5725dc49c02b19a7e075cd4996567c322416e6cc you can open another PR. This PR should only address the postinstall script issue.

**Comment by octref** [comment]
These all needs to be updated:

image

**Comment by yoyo930021** [comment]
@octref I'm done.

**Comment by octref** [comment]
LGTM, thanks 👍

**Pull request merged**